spec/factories/bulkrax_importers.rb

```diff
@@ -37,6 +37,7 @@
     limit { 10 }
     parser_fields { { 'import_file_path' => 'spec/fixtures/csv/good.csv' } }
     field_mapping { {} }
+    after :create, &:current_run
```

Comment:
@orangewolf Would it be better to move this to the model? Some kind of `after` hook that automatically creates a Run object after an Importer or Exporter is created instead of manually invoking `current_run` throughout the code?

Reply:
We have to do it on purpose. We create one during the ImportJob, and then pass it through the various import step.  Current_run makes a new record each session of runs.